src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const mult = days * 40;
```

Comment:
Good job! 

let's add some constant variables:
**[checklist] 2. Don't use literals in expressions (HARDCODE). Instead use constants explaining this values**

Reply:
Good point